web/libs/editor/src/components/HtxTextBox/HtxTextBox.jsx

```diff
@@ -1,6 +1,10 @@
 import React from "react";
-import { Tooltip, Typography } from "antd";
-import { DeleteOutlined, EditOutlined, EnterOutlined } from "@ant-design/icons";
+import { Typography } from "antd";
+import { Tooltip } from "../../common/Tooltip/Tooltip";
+import { EnterOutlined } from "@ant-design/icons";
+import { IconEdit, IconTrashAlt } from "../../assets/icons";
+import { Button } from "../../common/Button/Button";
+import { Elem } from "../../utils/bem";
```

Comment:
```suggestion
import { Typography } from "antd";
import { EnterOutlined } from "@ant-design/icons";
import { IconEdit, IconTrashAlt } from "../../assets/icons";
import { Button } from "../../common/Button/Button";
import { Tooltip } from "../../common/Tooltip/Tooltip";
import { Elem } from "../../utils/bem";
```

Reply:
ahh linter is not catching this? what do we need to do to get linter to handle this instead of manually updating this?